MANIFEST.in

```diff
@@ -1 +1,2 @@
 include README.rst
+prune workalendar/tests*
```

Comment:
I wasn't sure I could still run the tests effectively, but I may try to remove the test directory from the package, right.

respectfully, I can't see any use-case where you would need to import test packages at runtime. If you "pip install" stuff, you probably just want the part that works. Tests are just meant to be imported / loaded in a test env, but they're probably not a *runtime* package.

```python
>>> from workalendar.tests.test_europe import FranceTest
>>> # Then what?
```